cppclient/beast.h

```diff
@@ -24,13 +25,16 @@ namespace cppclient {
 class BeastIotaAPI : virtual public IotaAPI, public IotaJsonAPI {
  public:
   BeastIotaAPI() = delete;
-  BeastIotaAPI(std::string host, uint32_t port) : _host(std::move(host)), _port(port) {}
+  BeastIotaAPI(std::string host, uint32_t port, bool useSsl) : IotaAPI(useSsl), _host(std::move(host)), _port(port) {}
```

Comment:
snake_case

Reply:
This library was written with camelCase

Reply:
I was mean the `useSsl`, if the style is camelCase then should try to keep the consistency, in this case, `post_plain`, `post_ssl` need a change.

Reply:
correct, i overlooked that :)